device_doctor/lib/src/health.dart

```diff
@@ -88,14 +88,12 @@ class HealthCheckResult {
 Future<void> healthcheck(Map<String, List<HealthCheckResult>> deviceChecks) async {
   if (deviceChecks.isEmpty) {
     stderr.writeln('No healthy device is available');
-    throw StateError('No healthy device is available');
   }
   for (String deviceID in deviceChecks.keys) {
     List<HealthCheckResult> checks = deviceChecks[deviceID];
     for (HealthCheckResult healthCheckResult in checks) {
       if (!healthCheckResult.succeeded) {
         stderr.writeln('${healthCheckResult.name} check failed with: ${healthCheckResult.details}');
-        throw StateError('$deviceID: ${healthCheckResult.name} check failed with: ${healthCheckResult.details}');
```

Comment:
Why is this removed? How else does device_doctor indicate to the bot config the health checks failed?

Reply:
It is based on process.communicate, which uses stdout and stderr message.